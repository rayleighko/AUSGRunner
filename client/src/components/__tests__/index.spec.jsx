import React from 'react'
import {shallow} from 'enzyme'
import {
	Layout, Header, Footer, CreatedBy, Home, PlayGround, 
	BackToHome, Player, Ball, Map, BottomTile, WallTile
} from '../'


describe('Common Components', () => {
	describe('Layout test suite', () => {
		it('should render', () => {
			const tree = shallow(<Layout/>)
			expect(tree).toMatchSnapshot()
		})
	})

	describe('Header test suite', () => {
		it('should render', () => {
			const tree = shallow(<Header/>)
			expect(tree).toMatchSnapshot()
		})
	})

	describe('Footer test suite', () => {
		it('should render', () => {
			const tree = shallow(<Footer/>)
			expect(tree).toMatchSnapshot()
		})
	})
})

describe('Menu Components', () => {
	describe('CreatedBy test suite', () => {
		it('should render', () => {
			const tree = shallow(<CreatedBy/>)
			expect(tree).toMatchSnapshot()
		})
	})

	describe('Home test suite', () => {
		it('should render', () => {
			const tree = shallow(<Home/>)
			expect(tree).toMatchSnapshot()
		})
	})

	describe('PlayGround test suite', () => {
		it('should render', () => {
			const tree = shallow(<PlayGround/>)
			expect(tree).toMatchSnapshot()
		})
	})

	describe('BackToHome test suite', () => {
		it('should render', () => {
			const tree = shallow(<BackToHome/>)
			expect(tree).toMatchSnapshot()
		})
	})
})

describe('Creature Components', () => {
	describe('Player test suite', () => {
		it('should render', () => {
			const tree = shallow(<Player/>)
			expect(tree).toMatchSnapshot()
		})
	})
})

describe('Elements Components', () => {
	describe('Ball test suite', () => {
		it('should render', () => {
			const tree = shallow(<Ball/>)
			expect(tree).toMatchSnapshot()
		})
	})
})

describe('Map Components', () => {
	describe('Map test suite', () => {
		it('should render', () => {
			const tree = shallow(<Map/>)
			expect(tree).toMatchSnapshot()
		})
	})

	describe('BottomTile test suite', () => {
		it('should render', () => {
			const tree = shallow(<BottomTile/>)
			expect(tree).toMatchSnapshot()
		})
	})

	describe('WallTile test suite', () => {
		it('should render', () => {
			const tree = shallow(<WallTile/>)
			expect(tree).toMatchSnapshot()
		})
	})
})