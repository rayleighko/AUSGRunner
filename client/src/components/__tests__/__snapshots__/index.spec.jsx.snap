// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Common Components Footer test suite should render 1`] = `ShallowWrapper {}`;

exports[`Common Components Header test suite should render 1`] = `ShallowWrapper {}`;

exports[`Common Components Layout test suite should render 1`] = `ShallowWrapper {}`;

exports[`Creature Components Player test suite should render 1`] = `ShallowWrapper {}`;

exports[`Elements Components Ball test suite should render 1`] = `ShallowWrapper {}`;

exports[`Map Components BottomTile test suite should render 1`] = `ShallowWrapper {}`;

exports[`Map Components Map test suite should render 1`] = `ShallowWrapper {}`;

exports[`Map Components WallTile test suite should render 1`] = `ShallowWrapper {}`;

exports[`Menu Components BackToHome test suite should render 1`] = `ShallowWrapper {}`;

exports[`Menu Components CreatedBy test suite should render 1`] = `ShallowWrapper {}`;

exports[`Menu Components Home test suite should render 1`] = `ShallowWrapper {}`;

exports[`Menu Components PlayGround test suite should render 1`] = `ShallowWrapper {}`;
